cli/src/trusted_base_cli/mod.rs

```diff
@@ -104,7 +104,6 @@ fn new_account(trusted_args: &TrustedCli) -> CliResult {
 	drop(store);
 	info!("new account {}", key.to_ss58check());
 	let key_str = key.to_ss58check();
-	println!("{}", key_str);
```

Comment:
Maybe this leads to the CI error? Has this been changed for a reason?

Reply:
As we now give return values now, we did not need it, but we can reinsert it.